docs/guide/bundles.md

```diff
@@ -24,59 +24,102 @@ To learn more about organizing your eZ Platform project, see [Best Practices](be
+|[ezplatform-automated-translation](https://github.com/ezsystems/ezplatform-automated-translation)|adds automated translation|
+|[EzSystemsPrivacyCookieBundle](https://github.com/ezsystems/EzSystemsPrivacyCookieBundle)|adds privacy cookie banner|
+|[ezplatform-richtext](https://github.com/ezsystems/ezplatform-richtext)|Field Type for supporting rich formatted text stored in a structured XML format|
+|[CommentsBundle](https://github.com/ezsystems/CommentsBundle)|comment bundle for eZ Platform integrating with Disqus and Facebook and allowing custom integrations|
+|[EzSystemsRecommendationBundle](https://github.com/ezsystems/EzSystemsRecommendationBundle)|integration of YooChoose, a content recommendation solution|
+|[EzSystemsShareButtonsBundle](https://github.com/ezsystems/EzSystemsShareButtonsBundle)|adds social share buttons|
+|[RepositoryProfilerBundle](https://github.com/ezsystems/RepositoryProfilerBundle)|profiles Platform API/SPI and sets up scenarios to be able to continuously test to keep track of performance regressions of repository and underlying storage engines|
+|[QueryBuilderBundle](https://github.com/ezsystems/QueryBuilderBundle)|provides a PHP API dedicated to fluently writing repository queries|
+|[alloy-editor](https://github.com/ezsystems/alloy-editor)|WYSIWYG editor built on top of CKEditor, designed to create modern web content|
+|[ezmbpaex](https://github.com/ezsystems/ezmbpaex)|functionality for password expiration and validation|
+
+
+!!! Enterprise
+
+    |Bundle|Description|
+    |---------|-----------|   
+    |[ezstudio-cron](https://github.com/ezsystems/ezstudio-cron)|eZ Platform EE 1.x series only - exposes cron/cron package for use in eZ Platform (or just plain Symfony) via a simple command `ezpublish:cron:run`|
+    |[ez-service-network](https://github.com/ezsystems/ez-service-network)|provides API's and functionality for license handling and authentication needs|
+    |[cloud-deployment-manager](https://github.com/ezsystems/cloud-deployment-manager)|dedicated cloud deployment manager|
+    |[StudioUIBundle](https://github.com/ezsystems/StudioUIBundle)|contains the Studio editing interface in eZ Platform v1|
+    |[ezstudio-form-builder](https://github.com/ezsystems/ezstudio-form-builder)|Form Builder Bundle is a part of eZ Studio and is installed with eZ Studio meta repository in eZ Platform v1|
+    |[EzLandingPageFieldTypeBundle](https://github.com/ezsystems/EzLandingPageFieldTypeBundle)|Landing Page that is at the heart of StudioUI|
+    |[ezstudio-demo-bundle](https://github.com/ezsystems/ezstudio-demo-bundle)|represents a demo front-end website with eZ Studio|
+    |[ezstudio-personalized-block](https://github.com/ezsystems/ezstudio-personalized-block)|eZ Systems Personalized Block Bundle|
+    |[ezrecommendation](https://github.com/ezsystems/ezrecommendation)|allows customizing and personalizing your online portal individually for each end user|
```

Comment:
deprecated legacy extension. RecoBundle is relevant tough, unsure if I see it here.

Reply:
we have it under `EzSystemsRecommendationBundle`